spacy_transformers/architectures.py

```diff
@@ -47,6 +47,7 @@ def transformer_tok2vec_v1(
     name: str,
     get_spans,
     tokenizer_config,
+    transformers_config,
```

Comment:
As a quick note, if we want to add this parameter, it's probably best to bump the architecture version to `spacy-transformers.Tok2VecTransformer.v2` and update the documentation.

Reply:
Sure thing. I will take a look at it this or the next week.

Reply:
No pressure, this is just a maintenance detail!

Reply:
Not a problem, I have committed a new change. Hope it solves it. I considered also adding `spacy-transformers.TransformerModel.v1` unsure however if you want to split it up too much.

I assume the documentation updated depending on the docstring (i.e. I don't need to add documentation elsewhere?)

Reply:
Ah, the documentation is over at the `spacy` repo, but we can take care of that after the merge.